forms-html/Hussien-Neven/index.html

```diff
@@ -0,0 +1,60 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>form</title>
+</head>
+<body>
+    <form>
+        <label for="email">Enter Your email</label><br>
+        <input type="email" id="email" name="email" required><br>
+
+        <label for="username">Enter Your Username</label><br>
+        <input type="text" id="username" name="username"><br>
+        
+        <label for="password">Enter Your password</label><br>
+        <input type="password" id="password" name="password"><br>
+        
+        <label for="quantity">quantity between (1 - 5) :</label>
+        <input type="number" max="5" min="1" name="number" ><br>
+        
+        <label for="volume">Volume between (0 - 50) :</label>
+        <input type="range" id="volume"><br>
+        
+        <label for="radio">radio</label><br>
+        <input type="radio" id="male" name="gender" value="male"> 
+        <label for="male">Male</label><br>
+        <input type="radio" id="female" name="gender" value="female">
+        <label for="female">Female</label><br>
+        <input type="radio" id="other" name="gender" value="other">
+        <label for="other">Other</label><br>
+
+        <lable for="checkbox">checkbox</lable><br>
+        <input type="checkbox" id="checkbox" >I have a bike<br>
+        <input type="checkbox" id="checkbox" >I have a car<br>
+        <input type="checkbox" id="checkbox" >I have a boat<br>
+
+        <label for="button">button</label><br>
+        <input type="button" name="button" id="button" value="click me"><br>
+
+        <label for="color" name="color">select your favourite color: </label><br>
+        <input type="color" id="color"><br>
+
+        <label for="date"> enter a date before 01/01/1980 :</label><br>
+        <input type="date" name="date" id="date"><br>
```

Comment:
also here missing the validation, the input shouldn't accept any date after 1980-1-01